speakerCrtn.js

```diff
@@ -0,0 +1,32 @@
+const speakerGenerator = (object) => {
+// Function to go through data while
+// creating DOM elements needed
+  const card = document.createElement('div');
+  card.classList.add('speaker-card');
+  const containerD = document.createElement('div');
+  containerD.classList.add('speaker-card__img');
+  const cardImage = document.createElement('img');
+  cardImage.src = object.image;
+  cardImage.alt = `${object.name} profile picture`;
+  const containerE = document.createElement('div');
+  containerE.classList.add('speaker-card__text');
+  const titleDiv = document.createElement('h2');
+  titleDiv.classList.add('speaker-card__title');
+  titleDiv.innerText = object.name;
+  const paragraph1 = document.createElement('p');
+  paragraph1.classList.add('speaker-card__subtitle');
+  paragraph1.innerText = object.role;
+  const paragraph2 = document.createElement('p');
+  paragraph2.classList.add('speaker-card__bio');
+  paragraph2.innerText = object.bio;
```

Comment:
- [ ] You did a wonderful job creating all your elements dynamically :+1: , but, for semantics reasons, you should use an `article` tag for each element inside the "speakers" section

Reply:
@IngDavidTM what is the relevance of using the article tag in this section

Reply:
It is essential because the element represents a self-contained composition on the page intended to be independently distributable or reusable. [Here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article) you can find more information about it :+1:.